packages/codegen/src/schema.ts

```diff
@@ -63,6 +63,7 @@ export const revlinkSchema = z.object({
 export const tableSchema = z.object({
   name: z.string(),
   columns: z.array(columnSchema),
+  primaryKey: z.array(z.string()).optional(),
   revLinks: z.array(revlinkSchema).optional()
```

Comment:
Based on the comment below about potentially merging this into main instead of next https://github.com/xataio/client-ts/pull/1473#discussion_r1620885020 so that both legacy and new branches are supported, I'm a bit confused now about whether I should be removing things from this PR that are incompatible with legacy branches or trying to maintain compatibility

Reply:
Yeah, my comment was more on the hypothetic case if we would want to keep the data api option for a while (even on next channel). Right now if we merge this as-is we have some regressions (namely files). We can probably discuss it tomorrow in the project sync.